frontend-ssr/services/User.service.ts

```diff
@@ -0,0 +1,28 @@
+import MOCK_NOTIFICATIONS from '~/pages/api/notifications/mock';
+import MOCK_NOTIFICATION_READ from '../pages/api/notifications/mockAsRead';
+import MOCK_NOTIFICATIONS_ALL_READ from '../pages/api/notifications/mockAllAsRead';
+
+const LOAD_DURATION = 1000;
+
+class UserService {
+  static wait(mili) {
+    return new Promise((resolve) => setTimeout(resolve, mili));
+  }
+
+  static async getNotifications(page = 0) {
+    await UserService.wait(LOAD_DURATION);
+    return MOCK_NOTIFICATIONS;
+  }
+
+  static async markAsRead(id) {
+    await UserService.wait(LOAD_DURATION);
+    return MOCK_NOTIFICATION_READ;
```

Comment:
Not sure why this function and the below one are returning something. We don't expect/need any data when marking as read.

Reply:
Yes, you're right, if you remember I asked you a question about why we need 3 API's. In fact, I meant the 3 mocking files.